web/src/app/pages/create-survey/create-survey.component.html

```diff
@@ -13,23 +13,22 @@
   See the License for the specific language governing permissions and
   limitations under the License.
 -->
-<div>
-  <h1>Start building your survey</h1>
-  <form [formGroup]="formGroup">
-    <mat-form-field class="input" appearance="fill">
-      <mat-label>
-        Title
-      </mat-label>
-      <input matInput [formControlName]="titleControlKey" placeholder="Survey title" />
-    </mat-form-field>
 
-    <mat-form-field class="input" appearance="fill">
-      <mat-label>
-        Description
-      </mat-label>
-      <input matInput [formControlName]="descriptionControlKey" placeholder="Survey description" />
-    </mat-form-field>
-  </form>
-  <button id="back-button" mat-stroked-button (click)="back()">Back</button>
-  <button id="continue-button" mat-flat-button color="primary" (click)="continue()">Continue</button>
-</div>
\ No newline at end of file
+<div class="container" [ngSwitch]="setupPhase">
```

Comment:
Super nit, but why is ngSwitch on the container and not the card? Seems like it should be the immediate parent of the things being switched, at least for readability.